prophyc/options.py

```diff
@@ -73,6 +74,11 @@ def error(self, message):
                         type=readable_dir,
                         help='Generate C++ simple POD-based codec header and source files.')
 
+    parser.add_argument('--schema_out',
```

Comment:
Consider renaming `schema_out` to `prophy_out` -> for the sake of consistency. `x.prophy` file contains prophy language, `python_out` generates `x.py`, so `x.prophy` should be generated by `prophy_out`.

Also, putting this option below next or above previous option would allow to keep two C++ output variants next to each other (simpler for `--help` reader).